src/scripts/domain/game.ts

```
@@ -1,4 +1,4 @@
-import { Piece, PieceInfo } from 'domain/piece';
+import { Piece, PieceInfo, WallCell } from 'domain/piece';
```

ESLint

[error] 'WallCell' is defined but never used.

(@typescript-eslint/no-unused-vars)